polkadot/node/subsystem-bench/src/lib/statement/test_state.rs

```diff
@@ -130,6 +131,8 @@ impl TestState {
 				let mut receipt = receipt_templates[candidate_index].clone();
 				receipt.descriptor.set_para_id(Id::new(core_idx as u32 + 1));
 				receipt.descriptor.set_relay_parent(block_info.hash);
+				receipt.descriptor.set_core_index(CoreIndex(core_idx as u32));
+				receipt.descriptor.set_session_index(0);
```

Comment:
```suggestion
				receipt.descriptor.set_session_index(own_backing_group);
```

Reply:
Why does the session index need be set to to `own_backing_group` value ?